Gemfile

```diff
@@ -16,6 +16,7 @@ group :development, :unit_tests do
   gem 'simplecov',               :require => false
   gem 'puppet_facts',            :require => false
   gem 'json',                    :require => false
+  gem 'metadata-json-lint',      :require => false
```

Comment:
Removed line locally and retested successfully, so removed line from the commit.  Tests are successful.